README.md

```diff
@@ -147,6 +147,16 @@ npm run build:mf          # Builds source code within this project
 
 # Ongoing stuff
 
+# If this is not your first time using mock-firebolt but you want to ensure you have the latest SDK files you can use
+npm run upgradeSDK
+
+# After running the command to upgrade the SDK's you will need to run the build commands again
```

Comment:
Can we have "upgradeSDK" perform the builds? Less commands for folks to run.

Reply:
Added